ref/general/hardening-intro.adoc

```diff
@@ -22,49 +22,38 @@ Hardening is a general term that is used to describe the measures that are taken
-
-===== Use user registries that are safe for production
-Basic and Quick Start Security user registries are useful for development environments, but they are not acceptable for readiness testing, QA, or production use. Instead, use an LDAP user registry, federated repositories, or create a custom user registry. If you use an LDAP user registry, maintain your bind password in a separate `include` file, not in the `server.xml` file. If you use a custom registry, use the `UserRegistry` class to implement the registry, and make sure that the registry is thoroughly reviewed before it's used in production.
-
-===== Carefully configure file permissions
-When you configure file permissions, ensure that those who want access to the configuration fall into the appropriate permission bit. People who shouldn't have access at all fall into the `other` permission bit that's marked as `0`.
-
-===== Encrypt passwords
-Use encryption for passwords instead of Base64 encoding. You can use the `securityUtility` command, which provides plain text encryption for Open Liberty.
+* *Keep Open Liberty up to date*: To reduce operating system vulnerabilities, it's important to keep your version of Open Liberty up to date. Open Liberty fix packs are compatible with later versions, and the zero migration policy means that you can move to a newer fix pack without changing anything in your server configuration. Ensuring that you have a recent version of Open Liberty positions you for link:/docs/ref/general/#security-vulnerabilities.html[critical security fixes].
+* *Use user registries that are safe for production*: Basic and Quick Start Security user registries are useful for development environments, but they are not acceptable for readiness testing, QA, or production use. Instead, use an LDAP user registry, federated repositories, or create a custom user registry. If you use an LDAP user registry, maintain your bind password in a separate `include` file, not in the `server.xml` file. If you use a custom registry, use the `UserRegistry` class to implement the registry, and make sure that the registry is thoroughly reviewed before you use it in production.
+* *Carefully configure file permissions*: When you configure file permissions, ensure that the users who want access to the configuration fall into the appropriate permission bit. Users that are not the owner and are not part of the authorized group fall into the `other` permission bit, which is marked as `0`.
+* *Encrypt passwords*: Use encryption for passwords instead of Base64 encoding. You can use the `securityUtility` command, which provides plain text encryption for Open Liberty. For more information about the `securityUtility command`, go to <link pending>.
```

Comment:
need to double check with Ajay on the terminology here. In my user registry topic, he recommended using "hash encoding" as opposed to encrypting. They are two separate things. We should clarify with him which is best practice. I need to know as well and then make this section, my doc, and the forthcoming securityUtility doc consistent.

Reply:
Okay, agreed.